utilities/manual/source/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - repair user macro trigger (?highlight-as:...), still not usable for math env detection
 - add new user macro trigger (?inEnv:...)
 - add Grid menu to windowed and embedded pdf-viewer's context menu [#3942](https://github.com/texstudio-org/texstudio/pull/3942)
+- add "Set Column" (sets column of first page, same as Shift+Click) to Grid menu of pdf-viewer's context menu [#3974](https://github.com/texstudio-org/texstudio/pull/3974)
```

Comment:
the naming should match "set column" <-> "start column"
Logically it is "set columns offset" ? "set starting column" ?